membership.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Membership
+layout: post
+---
+
+## SKIP Student Association Structure Overview
+
+The SKIP Student Association operates in collaboration with Göta Studentkår and is designed to enhance the student experience by organizing events. To effectively achieve its mission, SKIP comprises two distinct categories of members: "Board Members" and "Regular Members." It's important to note that all members, both board and regular, must also be members of Göta Studentkår.
+
+![Skip structure](/assets/imgs/skip-structure.png)
+
+
+[comment]: <> (Be wary of whitespace after list items)
+1.
```

Comment:
I'd suggest to include the **membership type** after the ordered list items, just like in the template!